README.md

```diff
@@ -17,9 +17,10 @@ Storybook test runner turns all of your stories into executable tests.
     - [2. Running against locally built Storybooks in CI](#2-running-against-locally-built-storybooks-in-ci)
   - [Experimental test hook API](#experimental-test-hook-api)
     - [Image snapshot recipe](#image-snapshot-recipe)
+    - [Render lifecycle](#render-lifecycle)
   - [Troubleshooting](#troubleshooting)
-    - [The test runner seems flaky and keeps timing out](#the-test-runner-seems-flaky-and-keeps-timing-out)
-    - [Adding the test runner to other CI environments](#adding-the-test-runner-to-other-ci-environments)
+      - [The test runner seems flaky and keeps timing out](#the-test-runner-seems-flaky-and-keeps-timing-out)
+      - [Adding the test runner to other CI environments](#adding-the-test-runner-to-other-ci-environments)
```

Comment:
Was this change intended?
```suggestion
    - [The test runner seems flaky and keeps timing out](#the-test-runner-seems-flaky-and-keeps-timing-out)
    - [Adding the test runner to other CI environments](#adding-the-test-runner-to-other-ci-environments)
```

Reply:
I use a VSCode addon that indents the ToC hierarchically. I think it reads better, and am tweaking it more in a followup commit. Open to other conventions though!

Reply:
sounds good!